docs/GDI-SOP_sop-accessioning.md

```diff
@@ -109,13 +109,13 @@ The repository is divided in two main directories:
   - `docs/`: Contains all other supporting documents.
 
 ## SOP Life-cycle
-In this section we briefly describe the common life-cycle of an SOP within the GDI platform. For further details, take a look at the [**summary diagram**](../README.md#summary-diagram).
+In this section we briefly describe the common life-cycle of an SOP within the GDI platform. For further details, take a look at the [**summary diagram**](../README.md#summary-diagram) and the [SOP Template creation SOP](../sops/european-level/GDI-SOP0007_SOP-template-creation.md#7-summary-or-context-diagram).
 
 1. SOP is **requested** by any GDI member by making use of the [**"New SOP request"**](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/issues/new/choose) GitHub Issue Template.
-2. SOP Request is evaluated by the 1+MG Working Group, which approves or denies it.
+2. SOP Request is evaluated by the OC/SDPC, who approves or denies it.
 3. The SOP starts its development phase (content filling, review, approval...). This may have multiple steps, based on the chose initial method (e.g. Google Docs, GitHub...). Regardless of the method, the [general SOP template](GDI-SOP_sop-template.md) is used to construct it.
 4. The SOP gets to a level of maturity where it can be added through a Pull Request to the source GitHub repository after approval from authorizers (e.g. Management Board, 1+MG Working Group).
```

Comment:
Question: should we change the _Management Board_ and _1+MG Working Group_ to **OC/SDPC** here as well?

Reply:
I didn't get the question, @erikhedman - If I'm not mistaken I already replaced it with OC/SDPC in that document, right?

Or is the question to have it replaced in the fourth step, the one that speaks about authorizers?

Reply:
Ah, I just saw your other comment pinpointing to row 117. The authorizers will remain as the _Management Board_ and/or _1+MG Working Group_

It's just the approvers that got to review requests